bin/mysociety

```diff
@@ -144,18 +144,25 @@ case $COMMAND in
             if [ "$COMMAND" == "deploy" ]; then COMMAND=""; fi
             BALANCERS=$(mysociety vhost balancers "$VHOST")
             SERVERS=$(mysociety vhost servers "$VHOST")
+            SERVERS_INTERNAL=$(jq -r -c .vhosts.\"${VHOST}\".servers_internal[]? /data/vhosts.json 2>/dev/null)
```

Comment:
```suggestion
            SERVERS_INTERNAL=$(jq -r -c '.vhosts.\"${VHOST}\".servers_internal[]?' /data/vhosts.json 2>/dev/null)
```

Reply:
It isn't needed - that line was already present and working, I just moved it (from line 149). In fact, quoting it like that generates an error for me, so I'm not sure what's happening there!

Reply:
For the record, this is due to different shell quoting and variable interpolation approaches.